WebStoreApplication/Controllers/APIControllers/OrdersController.cs

```diff
@@ -33,13 +33,14 @@ public IActionResult GetOrder(int orderID)
             return Ok(dbAccessor.GetOrder(orderID));
         }
 
-        [HttpGet("{userID}")]
-        public IActionResult GetAllOrdersForUser(int userID)
+        [HttpGet("all/")]
+        public IActionResult GetAllOrderedItems()
         {
-            return Ok(dbAccessor.GetAllOrdersForUser(userID));
+            int userID = 2;
+            string statusDescription = "Accepted";
```

Comment:
Shouldn't userID and statusDescription be url parameters?

Reply:
Ah I forgot this was in here. statusDescription will stay as is for now though until finalizations can be made with regards to the changing of statuses

Reply:
Cart will always be the cart of the currently logged in user, that's why i get it from the session. statusDescription will stay hardcoded until changing statuses can be implemented or scrapped

Reply:
Oh I see. Cool Makes Sense.